src/annotator/integrations/pdf.js

```diff
@@ -61,14 +62,18 @@ export class PDFIntegration {
     this.annotator = annotator;
 
     const window_ = /** @type {HypothesisWindow} */ (window);
-    this.pdfViewer = window_.PDFViewerApplication.pdfViewer;
+
+    // Assume this class is only used if we're in the PDF.js viewer.
+    const pdfViewerApp = /** @type {PDFViewerApplication} */ (window_.PDFViewerApplication);
+
+    this.pdfViewer = pdfViewerApp.pdfViewer;
     this.pdfViewer.viewer.classList.add('has-transparent-text-layer');
 
     // Get the element that contains all of the PDF.js UI. This is typically
     // `document.body`.
-    this.pdfContainer = window_.PDFViewerApplication?.appConfig?.appContainer;
+    this.pdfContainer = pdfViewerApp.appConfig?.appContainer ?? document.body;
```

Comment:
The null-ness check for `window_.PDFViewerApplication` here was redundant. The `document.body` fallback is added for PDF.js versions before v1.5.188.